src/components/checkboxes/_macro.njk

```diff
@@ -2,9 +2,33 @@
     {% from "components/fieldset/_macro.njk" import onsFieldset %}
     {% from "components/mutually-exclusive/_macro.njk" import onsMutuallyExclusive %}
     {% from "components/checkboxes/_checkbox-macro.njk" import onsCheckbox %}
+    {% from "components/button/_macro.njk" import onsButton %}
 
     {% set fields %}
         <p class="ons-checkboxes__label{{ " " + params.checkboxesLabelClasses if params.checkboxesLabelClasses }}">{{ params.checkboxesLabel }}</p>
+        {% set hasOther = false %}
+        {% for checkbox in params.checkboxes %}
+            {% if checkbox.other is defined and checkbox.other %}
+                {% set hasOther = true %}
+            {% endif %}
+        {% endfor %}
+        {% if hasOther == false and params.autoSelect is defined and params.autoSelect and params.autoSelect.selectAllText is defined %}
```

Comment:
I think we need a `and params.autoSelect.selectAllText` in this if here

Reply:
I don't think we will need this because if `selectAllText` is provided it will always have a value